pkg/apis/sedna/v1alpha1/incrementallearningjob_types.go

```diff
@@ -96,6 +96,15 @@ type InitialModel struct {
 
 type DeployModel struct {
 	Name string `json:"name"`
+	// HotUpdateEnabled will enable the model hot update feature if its value is true.
```

Comment:
Please create a new pr, don't mix into one pr with docs update

Reply:
sorry about this mistake. I have fix the rebase error